benchmark/fluid/fluid_benchmark.py

```diff
@@ -296,9 +331,10 @@ def train_parallel(avg_loss, infer_prog, optimizer, train_reader, test_reader,
             if iters == args.skip_batch_num:
                 start_time = time.time()
                 num_samples = 0
-            if iters == args.iterations:
+            # NOTE: if use reader ops, the input data is not splited to multiple cards
+            if args.use_reader_op and iters >= args.iterations / args.gpus:
```

Comment:
I don't think `iters >= args.iterations / args.gpus` is appropriate. 
Because the model's accuracy is highly related to the new parameters that have learned, but the new parameters may be related to the times of updating parameter. So maybe we should not do that.

Reply:
Well, `args.iterations` is intended to let the benchmark finish fast, no concerns for model accuracy. To run a full model training, we can set `args.iterations` to -1 so that it can run until all train data have been fed.